WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -80,6 +83,10 @@ class PointOfSaleAggregateModel: ObservableObject, PointOfSaleAggregateModelProt
         publishOrderState()
         observeInternalOrderState()
         setupReaderReconnectionObservation()
+
+        if ServiceLocator.featureFlagService.isFeatureFlagEnabled(.sendReceiptsForPointOfSale) {
```

Comment:
I've just added the feature flag here to save a request to the system status store if not needed, but we can remove it if you feel is unnecessary, since the whole button is already under a feature flag 👍